cpp/daal/src/data_management/service_numeric_table.h

```diff
@@ -396,6 +398,75 @@ class SOANumericTableCPU : public SOANumericTable
         DAAL_DEFAULT_CREATE_TEMPLATE_IMPL_EX(SOANumericTableCPU, DAAL_TEMPLATE_ARGUMENTS(cpu), ddict, nRows, memoryAllocationFlag);
     }
 
+    template <typename T>
+    services::Status getTFeature(size_t feat_idx, size_t idx, size_t nrows, int rwFlag, BlockDescriptor<T> & block)
+    {
+        size_t nobs = getNumberOfRows();
+        block.setDetails(feat_idx, idx, rwFlag);
+
+        if (idx >= nobs)
+        {
+            block.resizeBuffer(1, 0);
```

Comment:
Is it necessary to change size of block here? If there are any other possibilities to invalidate it?